apps/platform-app/prisma/schema.prisma

```diff
@@ -99,6 +100,13 @@ model GitHubDetails {
   userId   String @unique
 }
 
+model LinkedInDetails {
+  id       String @id @default(uuid())
+  username String @unique
```

Comment:
**issue:** currently this LinkedInDetails gives us nothing - we should add anything that will establish relation between linkedIn and GDH. Can we add LinkedIn user ID here? I mean: `linkedInUserId String @unique`. It will enable to do something in the future with that connected accounts

Reply:
linkedin id (sub) is currently stored in `Accounts` model under `providerAccountId`, we cant move it as it is obligatory to make auth.js working correctly. should we store it twice then?

Reply:
hmm, now it looks like this LinkedInDetails got totally redundant (especially without this vanityName), as unique `username` could be a part of `User` model.